rackcity/views/rackcity_utils.py

```diff
@@ -83,13 +84,32 @@ def records_are_identical(existing_data, new_data):
             and new_data[key] != existing_data[key]
         ):
             if not (
-                isinstance(existing_data[key], int)
-                and int(new_data[key]) == existing_data[key]
+                int_string_comparison(existing_data[key], new_data[key])
+                or empty_string_null_comparison(existing_data[key], new_data[key])
             ):
                 return False
     return True
 
 
+def int_string_comparison(existing_value, new_value):
+    return (
+        isinstance(existing_value, int)
+        and isinstance(new_value, str)
+        and int(new_value) == existing_value
+    )
```

Comment:
this will throw a ValueError exception if new_value cannot be cast as an int (i.e. is "hi"). Is that the intended behavior?

Reply:
Just tested and this actually fails earlier with a good message because the validator would throw the error

Reply:
I love that for us